easybuild/easyconfigs/s/sisl/sisl-0.15.2-foss-2024a.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'sisl'
+version = '0.15.2'
+
+homepage = 'https://zerothi.github.io/sisl/'
+description = """sisl is an open-source easy-to-use density functional theory API
+ framework to post-analyse density functional theory codes output as well as
+ providing tight-binding calculation capabilities."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+builddependencies = [
+    ('scikit-build-core', '0.10.6'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('ASE', '3.23.0'),
+    ('mpi4py', '4.0.1'),
+    ('netCDF', '4.9.2'),
+    ('plotly.py', '5.24.1'),
+    ('scikit-image', '0.25.0'),
+    ('tqdm', '4.66.5'),
+    ('xarray', '2024.11.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('cftime', '1.6.4', {
+        'checksums': ['e325406193758a7ed67308deb52e727782a19e384e183378e7ff62098be0aedc'],
+    }),
+    ('nodify', '0.0.11', {
+        'checksums': ['b448c12fc637e890e743a8d2e1aec86e29e6cda5c30d6b7a4495461780dd90ac'],
+    }),
+    ('netCDF4', '1.7.2', {
```

Comment:
@appolloford can you use `netcdf4-python-1.7.1.post2-foss-2024a.eb` as a proper dependency instead?
or if you prefer/want/need `v1.7.2`, create an updated easyconfig for it, as it's not yet used as a dep?

Reply:
Thanks! I didn't notice this easyconfig. I will add it to the dependencies